### PR TITLE
Container updates

### LINKED ISSUE #529: Establish migration plan for new containers
## Description

Add new containers and establish a plan for migrating usages.

[Figma specs](https://www.figma.com/file/9Km09NjlZHYWsMP7EGT8tI/WIP-Sage-3-%E2%80%94-Admin-Components?node-id=3951%3A16287)

### PR DESCRIPTION
**Pull request by philschanely**
## Description

Closes #529 

This PR updates the container sizes to 3.0 specs and its [partner PR](https://github.com/Kajabi/kajabi-products/pull/19420) updates the sizes in `kajabi-products` as well.

[Figma specs](https://www.figma.com/file/9Km09NjlZHYWsMP7EGT8tI/WIP-Sage-3-%E2%80%94-Admin-Components?node-id=3951%3A16287)

| Size name | Old value | New value | Old size converts to new size |
|---|---|---|---|
| `tiny` | N/A | `200px` | N/A |
| `xs` | `760px` | `240px` | `lg` |
| `sm` | `900px` | `352px` | `xl` |
| `md` | `1200px` | `544px` | `xl` |
| `lg` | `1440px` | `736px` | `full` |
| `xl` | N/A | `1120px` | N/A |
| `full` | N/A | `1440px` | N/A |
| `fluid` | `100%` | Deprecated | `full` (default) |
| `modal` | `500px` | Deprecated | `md` |
| `modal-lg` | `900px` | Deprecated | Only use in `_modal.scss` is replace with `900px` manually until a full fix can be determined |

## Testing in `sage-lib`

Containers throughout the Documentation site are updated; a general browse through the site should reveal no regressions or bugs.

See updated Layout > Containers and Components > Container pages.

## Testing in `kajabi-products`
1. (BREAKING until [fix PR](https://github.com/Kajabi/kajabi-products/pull/19420) in `kajabi-products` merges; then MEDIUM) Adjusts settings behind container components used throughout `kajabi-products`. Minor changes in the size of containers is expected but no adverse effects. UXD to validate no regressions.

### CODE REVIEW

File: docs/app/views/application/_footer.html.erb [hunk 1]
```diff
@@ -1,39 +1,37 @@
 <div class="docs-footer">
-  <%= sage_component SageContainer, { size: "md" } do %>
```
Comment (philschanely) [inline docs/app/views/application/_footer.html.erb:2]: This container and similar instances on `_home...` partials that added extra inner `div`s are consolidated into one container in `docs/app/views/pages/index.html.erb`

File: docs/app/views/pages/container.html.erb [hunk 2]
```diff
@@ -1,143 +1,26 @@
...
+    <code><%= c[:alias] %></code>
+  <% end %>
+
+  <% t.column :max_size, label: "Maximum size" do |c| %>
+    <%= c[:max_size] %>
+  <% end %>
+
+  <% t.column :usage, label: "Usage" do |c| %>
+    <%= c[:usage] %>
+  <% end %>
```
Comment (joedakroub) [inline docs/app/views/pages/container.html.erb:22]: Much better! 🙌🏼

**Review by joedakroub — APPROVED** [review]
These changes look terrific! Thanks Phil! 👍🏼

File: docs/app/views/examples/components/container/_props.html.erb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<tr>
+  <td><%= md('`size`') %></td>
+  <td><%= md("Sets the maximum size for the container. See the <a href=\"#{pages_layout_path(:container)}\">Layout Containers</a> page.") %></td>
+  <td><%= md("`\"#{SageTokens::CONTAINER_SIZES.join("\"`, `\"")}\"`") %></td>
```
Comment (QuintonJason) [inline docs/app/views/examples/components/container/_props.html.erb:4]: YO! This is super slick!

File: packages/sage-assets/lib/stylesheets/components/_modal.scss [hunk 4]
```diff
@@ -96,7 +96,7 @@ $-modal-header-image-size: rem(28px);
   }
 
   .sage-modal--large & {
-    max-width: sage-container(modal-lg);
+    max-width: rem(900px); // TODO: Investigate this size necessity
```
Comment (QuintonJason) [inline packages/sage-assets/lib/stylesheets/components/_modal.scss:99]: Is this todo still valid?

Comment (philschanely) [reply packages/sage-assets/lib/stylesheets/components/_modal.scss:99]: yes; i didn't want it to hold up this PR and the modal sizing may rightfully be outside the scope of the layout containers. I did forget to add the ticket for the follow up though. I will add that now.

**Review by QuintonJason — APPROVED** [review]
This is some really nice work!

File: packages/sage-assets/lib/stylesheets/components/_modal.scss (hunk 4 above)
Comment (philschanely) [reply packages/sage-assets/lib/stylesheets/components/_modal.scss:99]: FYI #572

File: docs/app/views/examples/components/container/_props.html.erb (hunk 3 above)
Comment (monkeypox8) [reply docs/app/views/examples/components/container/_props.html.erb:4]: :fire: Seconding, this shortcut is incredible!

**Review by monkeypox8 — APPROVED** [review]
👍🏼 Beautifully done! Tested with the bridge and accompanying kp branch, looks great!

**Pull request merged**